.changeset/friendly-deers-glow.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+docs: Un-documented built-in binaries
```

Comment:
This PR is no longer only about docs. Since we'll now require `fuelup`, plus `forc` and `fuel-core`, we should probably treat this as a breaking change.

```suggestion
chore!: Remove built-in binaries for `forc` and `fuel-core`
```